zhaquirks/__init__.py

```diff
@@ -128,6 +131,11 @@ def _update_attribute(self, attrid, value):
             )
 
     def _calculate_battery_percentage(self, raw_value):
+        """Calculate battery percentage from voltage."""
+        # Required:    raw_value: int
```

Comment:
The required / optional comment doesn’t really apply. Let’s remove it.

Reply:
Fixed.